test-suites/cvehound

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
```

Comment:
Please add a copyright and license header statement.

Reply:
I'm not sure about copyright. I just added standard one from other scripts in the repository.

Reply:
The copyright should be under your name, and license typically LGPL v2.1+